src/Stryker.Core/Stryker.Core/Mutants/FsharpMutantOrchestrator.cs

```diff
@@ -11,7 +12,7 @@
 namespace Stryker.Core.Mutants
 {
     /// <inheritdoc/>
-    public class FsharpMutantOrchestrator : BaseMutantOrchestrator<FSharpList<SynModuleOrNamespace>>
+    public class FsharpMutantOrchestrator : BaseMutantOrchestrator<FSharpList<SynModuleOrNamespace>, SemanticModel>
```

Comment:
We shouldn't introduce Roslyn types to the fsharp orchestrator

Reply:
I've switched to object as a placeholder type (as fsharp doesn't yet use the semantic model)